### ISSUE TITLE
Some letters are missed from transliteration

### ISSUE THREAD

**Issue by kapooostin** [malexer/SublimeTranslit#6]
I created Rus-Lat transliteration based on [the latest system for passports](https://en.wikipedia.org/wiki/Romanization_of_Russian#After_2013).

[Here is my dictionary](https://gist.github.com/kapooostin/7dee611febb02d11ae7fb25c2d2666de)

It doesn't change all the `e`s in transliterated texts.

Maybe it is because in this system `ё`, `э` и `е` should be converted to same character `е`.

**Comment by malexer** [comment]
Can you provide a small example of: source word (text), expected and actual result?

Also, would you mind if I add your dictionary file to the next version of SublimeTranslit?

**Comment by kapooostin** [comment]
```javascript
source: 'привет'
expected: 'privet'
result: 'privеt'
```

> Also, would you mind if I add your dictionary file to the next version of SublimeTranslit?

No, I don't mind. That would be great!

**Comment by malexer** [comment]
I have fixed this issue and new version (1.1.3) is available for update.
Also, I've included your table (Passport 2013) into SublimeTranslit.

Keep in mind that in order to fix the case when multiple Cyrillic characters are transliterated to the same Latin one (`еёэ` -> `e`), I've changed the order of mapping in configs.
So, items in "chars_mapping" should be defined now as `"cyrillic char": "latin char"`.
i.e.:
```
"chars_mapping":
{
    "А": "A",
    "Б": "B",
    "В": "V",
    ...
```

Please, let me know if your issue is solved in version 1.1.3

**Comment by kapooostin** [comment]
It works fine. Thank you!
